src/libostree/ostree-bootloader-zipl.c

```diff
@@ -78,8 +86,206 @@ _ostree_bootloader_zipl_write_config (OstreeBootloader  *bootloader,
   return TRUE;
 }
 
+static gboolean
+_ostree_secure_execution_get_keys (GPtrArray **keys,
+                                   GCancellable *cancellable,
+                                   GError **error)
+{
+  g_auto (GLnxDirFdIterator) it = { 0,};
+  if ( !glnx_dirfd_iterator_init_at (-1, SECURE_EXECUTION_HOSTKEY_PATH, TRUE, &it, error))
+    return glnx_prefix_error (error, "s390x SE: looking for SE keys");
```

Comment:
Is `/etc/se-hostkeys/` 100% required on a host? I'm doing some testing on s390x FCOS now and this is causing issues:

```
[root@cosa-devsh ~]# journalctl -b -1 -u ostree-finalize-staged.service 
May 23 01:55:26 cosa-devsh systemd[1]: Finished ostree-finalize-staged.service - OSTree Finalize Staged Deployment.
May 23 01:55:32 cosa-devsh systemd[1]: Stopping ostree-finalize-staged.service - OSTree Finalize Staged Deployment...
May 23 01:55:32 cosa-devsh ostree[1599]: Finalizing staged deployment
May 23 01:55:32 cosa-devsh ostree[1599]: Copying /etc changes: 7 modified, 0 removed, 33 added
May 23 01:55:32 cosa-devsh ostree[1599]: Copying /etc changes: 7 modified, 0 removed, 33 added
May 23 01:55:34 cosa-devsh ostree[1599]: error: Final bootloader swap: s390x SE: looking for SE keys: opendir(/etc/se-hostkeys/): No such file or directory
May 23 01:55:34 cosa-devsh systemd[1]: ostree-finalize-staged.service: Control process exited, code=exited, status=1/FAILURE
May 23 01:55:34 cosa-devsh systemd[1]: ostree-finalize-staged.service: Failed with result 'exit-code'.
May 23 01:55:34 cosa-devsh systemd[1]: Stopped ostree-finalize-staged.service - OSTree Finalize Staged Deployment.
May 23 01:55:34 cosa-devsh systemd[1]: ostree-finalize-staged.service: Consumed 2.089s CPU time.
```